README.md

```diff
@@ -38,3 +38,36 @@ If you need to build your documentation on a particular Julia version, you can i
 ```
 
 as the first entry after `steps:`.
+
+### Prefixing the Julia command
+
+In some packages, you may want to prefix the `julia` command with another command, e.g. for running tests of certain graphical libraries with `xvfb-run`.
+In that case, you can add an input called `prefix` containing the command that will be inserted to your workflow:
+
+```yaml
+      - uses: julia-actions/julia-docdeploy@v1
+        with:
+          prefix: xvfb-run
+```
+
+If you only want to add this prefix on certain builds, you can [include additional values into a combination](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#example-including-additional-values-into-combinations) of your build matrix, e.g.:
```

Comment:
I think it's less common that you want to build the documentation for multiple operating systems?

Reply:
True, I just copied this from -runtest without thinking too much :D